src/samples/EducationStart/index.tsx

```diff
@@ -262,6 +272,16 @@ const EducationStartCase: FunctionComponent<any> = () => {
 
   if (!isLoggedIn || shuttered === null) {
     return null;
+  } else if (currentDisplay === 'servicenotavailable') {
+    return (
+      <>
+        <AppHeader appname={t('EDUCATION_START')} hasLanguageToggle={false} />
```

Comment:
hasLanguageToggle should be true, Please check once.

Reply:
Yes it should be until unless the another PR from akshay's merge, I need to make it true once after his PR gets merged probably.

Reply:
I have removed this option as its been handled from sdk config